language/en-GB/en-GB.page_technical_requirements.ini

```diff
@@ -41,6 +41,6 @@ TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_SOFTWARE="Software"
 TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_SUPPORTED_DATABASES="Supported Databases"
 TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_HEADING_SUPPORTED_WEB_SERVERS="Supported Web Servers"
 TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_MYSQL_REQUIREMENTS_FOOTNOTE="(InnoDB support required)"
-TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_PHP_REQUIREMENTS_FOOTNOTE="(Magic Quotes GPC, MB String Overload = off)<br>(Zlib Compression Support, XML Support, INI Parser Support, JSON Support, Mcrypt Support, MB Language = Default)"
+TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_PHP_REQUIREMENTS_FOOTNOTE="(Magic Quotes GPC, MB String Overload = off)<br>(Zlib Compression Support, XML Support, INI Parser Support, JSON Support, MB Language = Default)"
```

Comment:
```suggestion
TECHNICAL_REQUIREMENTS_SUPPORT_TABLE_PHP_REQUIREMENTS_FOOTNOTE="(Magic Quotes GPC, MB String Overload = off)<br>(Zlib Compression Support, XML Support, INI Parser Support, JSON Support, Mcrypt Support (<3.8.2), MB Language = Default)"
```

Reply:
Since Mcrypt is flat out removed as a requirement, there's no need to list it here.  Maybe in the linked https://docs.joomla.org/Special:MyLanguage/Optional_Technical_Requirements page this version can be referenced.  Otherwise, everything in this table is intended to be hard system requirements that execution fails without, and as the alert notes the table is only applicable to latest release (so we aren't listing a table for everything 3.0 thru 3.2, then 3.3 thru whenever the Mcrypt requirement was fully decoupled, etc. etc.).